src/Nethermind/Nethermind.Core/Crypto/KeccakCache.cs

```diff
@@ -188,7 +186,7 @@ private struct Entry
         /// </summary>
         public const int Size = 128;
 
-        private const int PayloadStart = 8;
+        private const int PayloadStart = sizeof(uint);
```

Comment:
The alignment @benaadams to leverage that with `Address`. I was thinking about it :-)

Reply:
Switched the orders of the compare and copy 🙂

Reply:
This has been noticed 😅